docs/reference/pvc-finalizer-downgrade-issue.md

```diff
@@ -0,0 +1,32 @@
+* TOC
+{:toc}
+
+---
+title: Kubernetes Downgrade issue from 1.10 to 1.9 due to PV/PVC Protection
+---
+
+## PV/PVC Protection in Kubernetes 1.10
+
+When enabled, [PV/PVC Protection](docs/admin/admission-controllers.md#storage-object-in-use-protection-beta) prevents PV/PVC from being removed when the finalizers are removed. 
+
+## Downgrading issue
+
+After downgrading from Kubernetes 1.10 to 1.9, PV/PVCs that are created in Kubernetes 1.10 with PVC Protection cannot be removed.
+
+## Workaround
+
+Currently PV/PVC finalizers have to be manually removed so PV/PVC can be removed after downgrading to Kubernetes 1.9.
+
+Before downgrading to Kubernetes 1.9, disable `StorageObjectInUseProtection` plugin and restart admission controller.
```

Comment:
How to `disable `StorageObjectInUseProtection` plugin and restart admission controller`? I don't read the details of the doc `PV/PVC Protection` referenced above. If the approach is given in the doc, it'd be good to mention this. If not, it'd be great to give some details or reference some doc here.

Reply:
Just notice that @Bradamant3 already mentioned this in https://github.com/kubernetes/website/pull/7731#discussion_r174333735 :)